WordPress/src/main/java/org/wordpress/android/models/Person.java

```diff
@@ -1,33 +1,74 @@
 package org.wordpress.android.models;
 
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.wordpress.android.util.AppLog;
+
 public class Person {
-    public long personID;
+    private long personID;
+    private int localTableBlogId;
 
     private String username;
     private String firstName;
     private String lastName;
     private String displayName;
-    private String imageUrl;
+    private String avatarUrl;
     private Role role;
 
     public Person(long personID,
+                  int localTableBlogId,
                   String username,
                   String firstName,
                   String lastName,
                   String displayName,
-                  String imageUrl,
+                  String avatarUrl,
                   Role role) {
         this.personID = personID;
+        this.localTableBlogId = localTableBlogId;
         this.username = username;
         this.firstName = firstName;
         this.lastName = lastName;
         this.displayName = displayName;
-        this.imageUrl = imageUrl;
+        this.avatarUrl = avatarUrl;
         this.role = role;
     }
 
+    public static Person fromJSON(JSONObject json, int localTableBlogId) {
+        if (json == null) {
+            return null;
+        }
+
+        // Response parameters can be found in https://developer.wordpress.com/docs/api/1.1/get/sites/%24site/users/%24user_id/
+        try {
+            long personID = Long.parseLong(json.getString("ID"));
+            String username = json.optString("login");
+            String firstName = json.optString("first_name");
+            String lastName = json.optString("last_name");
+            String displayName = json.optString("nice_name");
+            String avatarUrl = json.optString("avatar_URL");
+            // We don't support multiple roles, so the first role is picked just as it's in Calypso
+            Role role = Role.fromKey(json.optJSONArray("roles").optString(0));
+
+            return new Person(personID, localTableBlogId, username, firstName, lastName, displayName, avatarUrl, role);
+        } catch (JSONException e) {
+            AppLog.e(AppLog.T.PEOPLE, "JSON exception occurred while parsing the user json: " + e);
+        } catch (NumberFormatException e) {
+            AppLog.e(AppLog.T.PEOPLE, "The ID parsed from the JSON couldn't be converted to long: " + e);
+        }
+
+        return null;
```

Comment:
This is not a remark per se but, perhaps it could be useful to add the `@Nullable` annotation to the method to "nudge" the caller to check for NPE. Not a biggie nor a blocker and totally up to you to add it or not. Cheers!